imported/w3c/web-platform-tests/preload/modulepreload.html is flaky failure

rniwa opened the pull request:
#### 157fba03ce157d1f62b97dbf55ddacc9e0cf6071
<pre>
imported/w3c/web-platform-tests/preload/modulepreload.html is flaky failure
<a href="https://bugs.webkit.org/show_bug.cgi?id=252888">https://bugs.webkit.org/show_bug.cgi?id=252888</a>

Reviewed by Youenn Fablet.

The flakiness is caused by dummy.js getting cached by preceding tests.
Fix the test by making its URL unique.

* LayoutTests/imported/w3c/web-platform-tests/preload/modulepreload.html:

Canonical link: <a href="https://commits.webkit.org/260810@main">https://commits.webkit.org/260810@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c1cb274ec9b6adf90654b860966bba2f227e6412

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/109442 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/18566 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/42204 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/940 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118606 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/20032 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9763 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/101718 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/115198 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/14927 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/98147 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/43124 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96899 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29800 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/84880 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/11310 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/31143 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/11974 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/8079 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/17340 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50747 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/13705 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/4083 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

youennf reviewed (approved):
Can you link to the corresponding WPT PR?

rniwa committed:
imported/w3c/web-platform-tests/preload/modulepreload.html is flaky failure
https://bugs.webkit.org/show_bug.cgi?id=252888

Reviewed by Youenn Fablet.

The flakiness is caused by dummy.js getting cached by preceding tests.
Fix the test by making its URL unique.

* LayoutTests/imported/w3c/web-platform-tests/preload/modulepreload.html:

Canonical link: https://commits.webkit.org/260810@main
